test/functionalTest/harnessFunctions.sh

```diff
@@ -90,49 +90,45 @@ function dMsg()
 
 # ------------------------------------------------------------------------------
 #
-# dbInit - 
+# brokerStart CB - 
```

Comment:
Probably when I started to develop the test was with an old version.
I have proved with the file that is on the  develop branch, and it works, so I  have restored the same file.

Fixed in 0a207f6